src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -166,7 +123,7 @@ private void openWindow(Stage mainStage) {
 
         for (ParserResult pr : failed) {
             String message = Localization.lang("Error opening file '%0'.", pr.getFile().get().getName()) + "\n"
-                    + pr.getErrorMessage();
+                             + pr.getErrorMessage();
```

Comment:
The rest of the code should also be moved to run async because it relies on opening the last opened files.